CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+# Version 3.3.0-BETA
+* Update OpenTelemetry to 1.14 [#2257](https://github.com/microsoft/ApplicationInsights-Java/pull/2257).
```

Comment:
we have only updated to 1.14-SNAPSHOT so far (the fork versioning is confusing sorry)
```suggestion
* Update OpenTelemetry to 1.13 [#2257](https://github.com/microsoft/ApplicationInsights-Java/pull/2257).
```

Reply:
https://github.com/microsoft/ApplicationInsights-Java/pull/2257/files#:~:text=val%20otelInstrumentationAlphaVersion%20%3D%20%221.14.0.1%2Dalpha%22 

it's 1.14 though?

Reply:
1.14 instrumentation hasn't been released yet

Reply:
1.14.0.1-alpha, i'm confused. does it still get counted as 1.13 since it's alpha? should we say 1.14.0.1-alpha instead?